### PR TITLE
dev-libs/xqilla: version bump to 2.3.4

### PR DESCRIPTION
**Pull request by jok-ts**
Closes: https://bugs.gentoo.org/672886

dev-libs/xqilla-2.3.3 doesn't compile against dev-libs/xerces-c-3.2.2-r1 which just went stable everywhere. 
Version bump to a version that does, only the subslot and the xerces-c dependency have been changed.

### CODE REVIEW

**Comment by juippis** [comment]
This package is masked, 
https://github.com/gentoo/gentoo/commit/5d4004106464e9a4687779511f211e692424f316

have you taken these into account (I see it at least doesn't depend on htmltidy anymore). Would you like to become the dedicated maintainer for it? 

Ping @mgorny

File: dev-libs/xqilla/metadata.xml [hunk 1]
```diff
@@ -7,6 +7,8 @@
 			certain tasks</flag>
 		<flag name="htmltidy">Use <pkg>app-text/htmltidy</pkg> when parsing
 			HTML</flag>
+		<flag name="tidy">Use <pkg>app-text/tidy-html5</pkg> when parsing
```
Comment (mgorny) [inline dev-libs/xqilla/metadata.xml]: This is a global flag, so  you don't need to describe it again.

File: dev-libs/xqilla/xqilla-2.3.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		$(use_enable static-libs static)
+}
+
+src_compile() {
+	default
+	use doc && emake docs devdocs
+}
+
+src_install () {
+	use doc && HTML_DOCS=( docs/{dev-api,dom3-api,simple-api} )
```
Comment (mgorny) [inline dev-libs/xqilla/xqilla-2.3.4.ebuild]: You can make the var `local`.

File: dev-libs/xqilla/xqilla-2.3.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	default
+
+	if ! use static-libs; then
```
Comment (mgorny) [inline dev-libs/xqilla/xqilla-2.3.4.ebuild]: Do you really need `.la` files for static libs?

Comment (jok-ts) [reply dev-libs/xqilla/xqilla-2.3.4.ebuild]: Probably not, I just tried to do this with minimal changes to the existing ebuild.

Comment (jok-ts) [reply dev-libs/xqilla/xqilla-2.3.4.ebuild]: > Please also include taking over maintenance and removing the p.mask (in separate commits).

I don't think I want to take over the maintenance.

**Comment by mgorny** [comment]
I'm afraid in this case I don't see the purpose of merging this as primary last rite reason (lack of maintainer) still stands. Merging it would be just patch work that would result in some other issue setting up back for removal later on.

**Comment by jok-ts** [comment]
- [ ] Unmaintained.  
- [x] Fails to build.  
- [x] Last bumped in 2016, and no new releases since.  
- [x] Still depends on dead app-text/htmltidy (#671442).

Well, got three out of four sorted. I'll close this PR later.

**Comment by jok-ts** [comment]
Closing after package cleanup.

**Pull request closed without merging**